### ISSUE TITLE
Initial scroll on reading

### ISSUE THREAD

**Issue by talkingseedling** [readrops/Readrops#184]
Version: Both 1.3.1 and 2.0beta01
Account: FreshRSS

When you first open an article, touching anywhere causes a jump scroll. It does not happen again with the same article open. Closing the current and opening any (including the same) article and you can recreate the jump.

To recreate: 
1. Touch on an article in the timeline to open it, it is presented in full.
2. Touch like you are about to swipe to scroll, the screen will jump scroll about a quarter of the screen.
3. Scroll back up to the top. Touch anywhere and you get the expected behaviour.

If you long hold after first opening an article, it'll first jump scroll, then try to highlight text back in the original location you touched, about an inch from your finger.

**Comment by Shinokuni** [comment]
Yeah, I've found out this issue too, a bit annoying, but not very blocking. It might be a kind of conflict between the webview rendering the article content and the page container.
